SerialPrograms/Source/PokemonSV/Programs/TeraRaids/PokemonSV_TeraRoller.cpp

```diff
@@ -114,6 +114,16 @@ TeraRoller::TeraRoller()
           LockWhileRunning::UNLOCKED,
           false
     )
+    , EVENT_CHECK_MODE(
+        "<b>Event Tera Raid Action:</b><br>Choose how to interact with event/non-event raids.",
+        {
+            {EventCheckMode::CHECK_ALL,             "check_all",        "Check all raids / No ongoing tera raid event"},

```

Comment:
Why specify no ongoing tera raid event ? Would the "only non event" break in that case ?

Reply:
Updated the wording for more clarity on which option the user should use